src/components/integrated/OakLessonLayout/OakLessonLayout.tsx

```diff
@@ -86,11 +86,7 @@ export const OakLessonLayout = ({
             {topNavSlot}
           </OakBox>
         )}
-        <OakFlex
-          $flexGrow={1}
-          $flexDirection="column"
-          $mh={["space-between-s", "space-between-l"]}
```

Comment:
This was interfering with the layout for the lesson overview — at the mobile breakpoint the subject icon and lesson name become a header to the content. The bottom border runs to the edge of the viewport which isn't possible with the original implementation (unless I employed negative margins 😉 )

<img width="783" alt="Screenshot 2024-01-31 at 17 46 44" src="https://github.com/oaknational/oak-components/assets/122096/94926e07-87d7-4ed6-bb86-422549b751ad">

This also interfered with the grid since the columns would be inset my the padding.

Easier to remove the automatic padding and leave it up to the consumer to apply a proper grid.

I'll go back and correct things where the layout has already been used and is reliant on the padding